### PR TITLE
Remove tech storage access from roboticists

### PR DESCRIPTION
**Pull request by variableundefined**
Remove roboticist's tech storage access to prevent them stealing boards and insulated gloves. They really have no business doing that. Ask someone with more access.

🆑:
tweak: Roboticists no longer have tech storage access.
/🆑

### CODE REVIEW

**Comment by TDSSS** [comment]
The RD also has access to tech storage since Roboticists are under him, right? Did yuo remove his access, too?

**Comment by ID404-NotFound** [comment]
If roboticists NEED things from tech storage, they can just ASK like normal people.

"interaction? in MY spess? preposterous!"

**Comment by Taaketa** [comment]
This reeks of 'i don't want them there.'

If the gloves are an issue then stick an engineer locker in there and put them in it. Should engineers have their access revoked because they have no business with having access to flashes?

**Comment by davidchan13** [comment]
Remove the flashes, intellicard and spare protolathe, circuit printer, borg charger and mech charger borgs to RnD server or similar secure science location then.

**Comment by taukausanake** [comment]
Maybe just get rid of the tech storage? You're going to RnD most of it anyways so what does it matter?

Unless of course those parts are _how_ you RnD

**Comment by Citinited** [comment]
> They have access also to all the tools they need to just go through the wall and take the batteries, flashes, board and gloves anyway.

Based on that we should give engineers and anyone who can print out a full set of tools all-access from round-start

**Comment by Denthamos** [comment]
A lot of the people commenting here seem to forget the fact tech storage is there for engineers, new build projects, and rebuilding areas of the station that have been destroyed.  The RD has access due to them being in charge of ALL technology on the station along with it being a research station, so it makes sense for them.

@Spartan6 So because anyone has the tools to go through the wall they should have access then?  Cargo access to tech storage?  Any assistant?  The argument for flashes and cells make no sense given given they can mass produce them at materials costs that are very minimal.  The cost for them is pretty much neglible given miners bring ANYTHING up along with the extra materials that are in EVA and arrivals aux storage. 

 They need gloves?  They can ask cargo and place in an order, because otherwise they have no need for insulated gloves as nothing in their job shocks them in regards to mechs/borgs/IPCs.

@davidchan13 In regards to the circuits that are in there, if engineering is building an entire own RnD lab of their own, that is a whole other issue which needs to either be handled IC wise or such in game.  Those boards are there incase for some reason, RnD and robotics as a whole gets blown up and there is nowhere to print new boards out.  Its a situation of not putting all your eggs in one basket.

Again, people forget the main purpose of tech storage is for _rebuilding_ and new projects.  Heavens forbid we encourage people to use the department for ordering supplies to order supplies, and make the department responsible for making new machines/parts/circuits actually use their setup for that which would take 5 more minutes of their time.  Hell there is a reason why on some servers tech storage is INSIDE engineering proper and does not have an airlock to the station common.

**Comment by TDSSS** [comment]
I disagree with the RD having access if Roboticists lose it. He's in charge of research, not 'technology'. The person responsible for running the technology on the station is the Chief Engineer.

**Comment by taukausanake** [comment]
> He's in charge of research, not 'technology'

This. However I would reword the next sentence to "installing" rather than "running" the technology. Science is in charge of operating or running the hardware. The engineers are in charge of getting it all put together

**Comment by variableundefined** [comment]
Closing down due to lack of support.
